sale_rental/models/stock.py

```diff
@@ -69,142 +68,138 @@ def _get_rental_push_pull_rules(self):
-                                'New view rental stock location created ID %d',
-                                view_loc.id)
-                        warehouse.rental_view_location_id = view_loc.id
-                    if not warehouse.rental_in_location_id:
-                        in_loc = slo.search(
-                            [('name', '=', 'Rental In'),
-                             ('location_id', '=',
-                                warehouse.rental_view_location_id.id)],
-                            limit=1)
-                        if not in_loc:
-                            in_loc = slo.create({
-                                'name': _('Rental In'),
-                                'location_id':
-                                warehouse.rental_view_location_id.id,
-                            })
-                            logger.debug(
-                                'New in rental stock location created ID %d',
-                                in_loc.id)
-                        warehouse.rental_in_location_id = in_loc.id
-                    if not warehouse.rental_out_location_id:
-                        out_loc = slo.search(
-                            [('name', '=', 'Rental Out'),
-                             ('location_id', '=',
-                                warehouse.rental_view_location_id.id)],
-                            limit=1)
-                        if not out_loc:
-                            out_loc = slo.create({
-                                'name': _('Rental Out'),
-                                'location_id':
-                                warehouse.rental_view_location_id.id,
-                            })
-                            logger.debug(
-                                'New out rental stock location created ID %d',
-                                out_loc.id)
-                        warehouse.rental_out_location_id = out_loc.id
-                    warehouse.write({
-                        'route_ids': [(4, rental_route.id)],
-                        'rental_route_id': rental_route.id,
-                        'sell_rented_product_route_id': sell_rented_route.id,
+                self._create_rental_locations()
+                self.write({
+                    'route_ids': [(4, rental_route.id)],
+                    'rental_route_id': rental_route.id,
+                    'sell_rented_product_route_id': sell_rented_route.id,
                     })
-                    for obj, rules_list in\
-                            self._get_rental_push_pull_rules().items():
-                        for rule in rules_list:
-                            self.env[obj].create(rule)
+                for rule_vals in self._get_rental_push_pull_rules():
+                    self.env['stock.rule'].create(rule_vals)
             else:
-                for warehouse in self:
-                    pull_rules_to_delete = self.env['procurement.rule'].search(
+                for wh in self:
```

Comment:
@alexis-via why changed `warehouse -> wh`?

Reply:
To have shorter variable names, therefore make it easier to respect the 79-chars per line rule